src/Shouldly.Tests/ShouldNotContain/StringContainsStringCaseIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldNotContain
+{
+    public class StringContainsStringCaseIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Shouldly is legendary".ShouldNotContain("legendary", Case.Insensitive);
```

Comment:
Yeah, `ShouldBe` makes sense too. I'll sort it. I presume this would be done using a custom message generator?